src/lib/services/chat/memories/langchain/chroma_store_retriever.py

```diff
@@ -189,3 +189,60 @@ def clear(self) -> 'LangChainChromaStoreMemory.Result':
             self.result.error_message = "No memory present"
             logger.error(self.result.error_message)
         return self.result
+
+    def save_message(self, message: Any) -> 'LangChainChromaStoreMemory.Result':
```

Comment:
Based on the function logic it seems the type hint could be changed from any to the specific expected types. `list[HumanMessage, AIMessage] | tuple[HumanMessage, AIMessage]`. 

Also I did notice in the test it looks like this is also potentially called where messages is a dictionary? If so also add `dict[str, HumanMessage | AIMessage]`